source/crd_test.go

```diff
@@ -383,7 +383,11 @@ func testCRDSourceEndpoints(t *testing.T) {
 			scheme := runtime.NewScheme()
 			addKnownTypes(scheme, groupVersion)
 
-			cs, _ := NewCRDSource(restClient, ti.namespace, ti.kind, ti.annotationFilter, ti.labelFilter, scheme)
+			// At present, client-go's fake.RESTClient (used by crd_test.go) is known to cause race conditions when used
+			// with informers: https://github.com/kubernetes/kubernetes/issues/95372
+			// So don't start the informer during testing.
+			startInformer := false
```

Comment:
It looks like there is a way around that. https://github.com/kubernetes/kubernetes/pull/95897